test/java/seedu/addressbook/parser/UtilsTest.java

```diff
@@ -0,0 +1,39 @@
+package seedu.addressbook.parser;
+
+import seedu.addressbook.common.Utils;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.*;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class UtilsTest {
+
+    @Test
+    public void isAnyNull_1NullObject_returnsTrue(){
+        Object obj1 = null;
+        try {
+            Address phone1 = new Address("123", false);
+            boolean hasNull = Utils.isAnyNull(obj1, phone1);
+            assertTrue(hasNull);
```

Comment:
Instead of handling exceptions inside the test, we can set the test method to throw the exception.